serverless/serverless-release-notes.adoc

```diff
@@ -11,10 +11,10 @@ For an overview of {ServerlessProductName} functionality, see xref:../serverless
 include::modules/serverless-getting-support.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
+include::modules/serverless-rn-1-7-1.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-6-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-5-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-4-0.adoc[leveloffset=+1]
```

Comment:
Do we have any rule about removing old releases from here? Just curious why 1.5.0 remains and 1.4.0 is being removed, for example.

Reply:
This is because 1.4.0 was for 4.2 and 4.3, but we've dropped 4.2 support now, so we should only include 1.5.0+ in the docs for 4.3+ :slightly_smiling_face: when this PR is merged, it will only be picked to 4.3, 4.4 and 4.5 docs, since we no longer update for 4.2 anyway.